console/billing/concepts.mdx

```diff
@@ -5,16 +5,143 @@ meta:
+
+</Concept>
+
+<Concept opened>
+
+## Billing contact
+
+The billing contact is an additional contact who will receive the invoice or payment alerts by email. This allows you to add a person who may not have access to the Scaleway console, but who should be aware of billing, payment documents and notifications.
+
+</Concept>
+
+<Concept opened>
+
+## Billing cycle
+
+Invoices are issued each month based on the resource consumption from the previous month. Our reference billing period is the definition of a calendar month in the Coordinated Universal Time (UTC) time zone. The automatic payment of invoices occurs in the first few days of the month, once the invoice is issued and processed using the preferred payment method of your account.
+
+</Concept>
+
+<Concept opened>
+
+## Billing information
+
+Consists of information about the organization that will be billed. They will appear on the feature and can be modified on the Scaleway console.
+
+</Concept>
+
+<Concept opened>
+
+## Consumption pie chart
+
+The consumption pie chart reflects the amount of money spent on the products you've used. Its monetary value is calculated in real-time, providing an instant overview of your expenses.
+
+</Concept>
+
+<Concept opened>
+
+## Cost Manager 
+
+Cost manager is a comprehensive and user-friendly tool to monitor and analyze cloud resource consumption, costs, and trends. It will offer a clear and detailed view of consumption data, enabling users to accurately reconcile costs for re-invoicing purposes.
+
+</Concept>
+
+<Concept opened>
+
+## Free Tier 
+
+The Free Tier offers certain products at no cost and up to a specified usage limit. These limits do not have an expiration date but may be subject to change. Eligible customers can enjoy these select services free of charge when creating an account, with a usage limit in place.
+
+</Concept>
+
+<Concept opened>
+
+## Free Trial
+
+The Free Trial provides users access to specific products at no cost. However, it is available only for a limited duration and usage quantity.
+
+</Concept>
+
+<Concept opened>
+
+## IAM billing permission
+
+Each group is linked to a permission set. There are two permission sets linked to billing:
+
+Billing read only: List and read access to billing information. This allows the user to view consumption, invoices, discounts, budget alerts and payment methods.
+Billing Manager: Full access to billing management. This allows the user to list, read and edit billing contact information, payment information, billing alerts and invoices.
+
+</Concept>
+
+<Concept opened>
+
+## Pay as you go 
+
+Public cloud services follow a 'pay as you go' model, ensuring you pay only for what you use. Some products offer plan options. Your monthly invoice, calculated at the end of the month, is based on exact resource usage and may include additional invoices for upfront payments during the month.
```

Comment:
I think it might be good to point out that Scaleway is one of these public cloud services?